sway/main.c

```diff
@@ -311,6 +326,20 @@ int main(int argc, char **argv) {
 	}
 	register_extensions();
 
+#ifdef __linux__
+	if (suid) {
+		// Drop every cap except CAP_SYS_PTRACE
+		cap_t caps = cap_init();
+		cap_value_t keep = CAP_SYS_PTRACE;
```

Comment:
No, it was required because a suid binary would not gain all caps as effective, and besides, all the privileged stuff is dealt with in a child process of wlc (forked in wlc_init, so it keeps everything), so it isn't.